internal/app/go-filecoin/node/init.go

```diff
@@ -146,14 +149,13 @@ func importInitKeys(w *wallet.Wallet, importKeys []*crypto.KeyInfo) error {
 }
 
 func ImportPresealedSectors(rep repo.Repo, srcPath string, sectorSize abi.SectorSize, symlink bool) error {
-	badgerOptions := badger.Options{
-		GcDiscardRatio: badger.DefaultOptions.GcDiscardRatio,
-		GcInterval:     badger.DefaultOptions.GcInterval,
-		GcSleep:        badger.DefaultOptions.GcSleep,
-		Options:        badger.DefaultOptions.Options,
-	}
-	badgerOptions.ReadOnly = true
-	oldMetaDs, err := badger.NewDatastore(filepath.Join(srcPath, "badger"), &badgerOptions)
+	nextSecnum, err := findNextSecnum(srcPath)
```

Comment:
Lotus's seed tool no longer provides a badger directory in the genesis-sectors. We were only using the badger database to set the sectornum, so we can do that directly. Checking the highest sectornum found in a file path works. Not this isn't production code.